requirements.txt

```diff
@@ -0,0 +1,3 @@
+anndata==0.10.3
+cellxgene-census==1.7.0
```

Comment:
I don't think this is required tbh, `cellxgene-census`

Reply:
I was just testing something in a notebook with it, will delete it.

Reply:
I think it will be important to have cellxgene-census in this lib.  It is currently our main stable source of AnnData files with resolvable IDs.  It will be used as such by integration efforts. The spec will support other sources, e.g. CAP.  More details to follow in general spec tickets.